clouddq/classes/dq_configs_cache.py

```diff
@@ -490,7 +473,7 @@ def _resolve_bigquery_entity_uri(
                 "Failed to retrieve Dataplex Metadata entry for "
                 f"entity_uri '{entity_uri.complete_uri_string}' "
                 f"with error:\n"
-                f"{pformat(dataplex_entities_match.json())}\n\n"
+                f"{pformat(json.dumps(dataplex_entities_match))}\n\n"
```

Comment:
do we need to add this fix to main branch first?

Reply:
This fix is already pushed to main.